go_utils/_cli.py

```diff
@@ -212,23 +258,53 @@ def lc_photo_download():
     parser.add_argument("input", help="Input Directory of the command")
     parser.add_argument("out", help="Output Directory of the command")
 
-    parser.add_argument("--up", "-u", help="Include Upward Photos", action="store_true")
-    parser.add_argument(
+    photo_group = parser.add_argument_group(
+        "Photo Types", "Includes: up, down, north, south, east, west"
+    )
+    photo_group.add_argument(
+        "--up", "-u", help="Include Upward Photos", action="store_true"
+    )
+    photo_group.add_argument(
         "--down", "-d", help="Include Downward Photos", action="store_true"
     )
-    parser.add_argument(
+    photo_group.add_argument(
         "--north", "-n", help="Include Northern Photos", action="store_true"
     )
-    parser.add_argument(
+    photo_group.add_argument(
         "--south", "-s", help="Include Southern Photos", action="store_true"
     )
-    parser.add_argument(
+    photo_group.add_argument(
         "--east", "-e", help="Include Eastern Photos", action="store_true"
     )
-    parser.add_argument(
+    photo_group.add_argument(
         "--west", "-w", help="Include Western Photos", action="store_true"
     )
-    parser.add_argument("--all", "-a", help="Include All Photos", action="store_true")
+    photo_group.add_argument(
+        "--all", "-a", help="Include All Photos", action="store_true"
+    )
+
+    name_group = parser.add_argument_group(
+        "Photo Naming", "Allows naming customization"
+    )
+    name_group.add_argument(
+        "--name_additional",
+        "-i",
+        help="Include additonal info in photo names.",
+        type=str,
+        action="store",
+    )
+
+    lc_name_fields = ["direction", "latitude", "longitude", "date_str", "lc_id"]
+    name_group.add_argument(
+        "--nargs_exclude",
+        "-x",
+        nargs="+",
+        help="Exclude these fields from photo names.",
+        choices=lc_name_fields,
+        type=str,
+        action="store",
+    )
+
```

Comment:
Again, see comment for photo url format string naming

Reply:
updated!